src/data/faq.json

```diff
@@ -535,8 +535,8 @@
                         "anchor": "close_app",
                         "active": true,
                         "textblock": [
-                            "Yes. As long as you have activated exposure logging in the app and Bluetooth is on, encounters are logged in the background. The app itself doesn't have to be open for this. You can minimize it or use 'swipe-to-quit' to close it.",
-                            "To be on the safe side, open the app once a day after 24 hours. You'll find more info at <a href='https://www.coronawarn.app/en/#howto' target='_blank' rel='noopener noreferrer'>This is how the app works best</a>.",
+                            "Yes. As long as you have activated exposure logging in the app and Bluetooth (and Location Services on Android) is on, encounters are logged in the background by the Exposure Notification system integrated in iOS/Android. The app itself doesn't have to be open for this. You can minimize it or use 'swipe-to-quit' to close it.",
+                            "To be on the safe side, open the app once a day. You'll find more info at <a href='https://www.coronawarn.app/en/#howto' target='_blank' rel='noopener noreferrer'>This is how the app works best</a>.",

```

Comment:
Suggest changing to relative reference
href='https://www.coronawarn.app/en/#howto' =>
href='../#howto'

Reply:
good idea!